packages/_private/vue-demo/src/DemoForm.vue

```diff
@@ -0,0 +1,323 @@
+        required
+        type="password"
+        v-model="formData.password"
+      />
+
+      <SynVueInput
+        id="input-number"
+        label="Please provide a fallback numeric value that may be used for password recovery"
+        min="1000"
+        max="9999"
+        name="code"
+        placeholder="Please choose a value with four digits, e.g. 1234"
+        type="number"
+        v-model="formData.code"
+      />
+    </DemoFieldset>
+    <!-- /Security -->
+
+    <SynVueDivider />
+
+    <!-- Topics -->
+    <SynFieldset legend="Topics">
+      <SynVueSelect
+        clearable
+        id="topics"
+        label="I am interested in the following technologies"
+        multiple
+        name="topics"
+        v-model="formData.topics"
+      >
+        <SynVueOptgroup label="Frontend">
+          <SynVueOption value="angular">Angular</SynVueOption>
+          <SynVueOption value="react">React.js</SynVueOption>
+          <SynVueOption value="vanilla">Vanilla.js</SynVueOption>
+          <SynVueOption value="vue">Vue</SynVueOption>
+        </SynVueOptgroup>
+        <SynVueOptgroup label="Backend">
+          <SynVueOption value="node">Node.js</SynVueOption>
+          <SynVueOption value="Python">Python</SynVueOption>
+        </SynVueOptgroup>
+      </SynVueSelect>
+    </SynFieldset>
+    <!-- /Topics -->
+
+    <SynVueDivider />
+
+    <!-- Marketing -->
+    <DemoFieldset legend="Please inform me about the following technologies">
+      <SynVueCheckbox
+        id="checkbox-newsletter-default"
+        name="newsletterDefault"
+        v-model="formData.newsletterStandard"
+      >
+        Please subscribe me to the synergy newsletter
+      </SynVueCheckbox>
+      <SynVueCheckbox
+        id="checkbox-newsletter-angular"
+        name="newsletterAngular"
+        v-model="formData.newsletterAngular"
+      >
+        Please subscribe me to all things related to angular
+      </SynVueCheckbox>
+      <SynVueCheckbox
+        id="checkbox-newsletter-react"
+        name="newsletterReact"
+        v-model="formData.newsletterReact"
+      >
+        Please subscribe me to all things related to react
+      </SynVueCheckbox>
+      <SynVueCheckbox
+        id="checkbox-newsletter-vanilla"
+        name="newsletterVanilla"
+        v-model="formData.newsletterVanilla"
+      >
+        Please subscribe me to all things related to vanilla.js
+      </SynVueCheckbox>
+      <SynVueCheckbox
+        id="checkbox-newsletter-vue"
+        name="newsletterVue"
+        v-model="formData.newsletterVue"
+      >
+        Please subscribe me to all things related to vue
+      </SynVueCheckbox>
+      <SynVueSwitch
+        id="checkbox-newsletter-beta"
+        name="newsletterBeta"
+        v-model="formData.newsletterBeta"
+      >
+        I am interested in the Synergy Beta Program
+      </SynVueSwitch>
+    </DemoFieldset>
+    <!-- /Marketing -->
+
+    <SynVueDivider />
+
```

Comment:
Additional Information section is missing

Reply:
How did I miss that?